### ISSUE TITLE
Wrong rule when sh script uses `disown` 

### ISSUE THREAD

**Issue by SpyrosRoum** [koalaman/shellcheck#2832]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or "online"): 0.9.0 
- [X] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [X] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [X] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/usr/bin/sh

prog & disown

```

#### Here's what shellcheck currently says:

> In ./<script> line 3:
prog & disown
        ^----^ SC3044 (warning): In POSIX sh, 'disown' is undefined.
>
> For more information:
  https://www.shellcheck.net/wiki/SC3044 -- In POSIX sh, 'disown' is undefined.

#### Here's what I wanted or expected to see:

The page it links to is for `declare`, not `disown`, and doesn't mention `disown` in any way. I couldn't find a page for `disown` but it's possible my search wasn't good..

**Comment by SpyrosRoum** [comment]
I took a look at the source code and notice that a lot of commands point to the same rule that's specific to declare. Perhaps editing a message to make it  more generic and mention the other commands would be sufficient.

List of commands that point to the same rule can be found [here](https://github.com/koalaman/shellcheck/blob/master/src/ShellCheck/Checks/ShellSupport.hs#L393-L397)

**Comment by brother** [comment]
The wiki is open for edits, please make it clearer!

**Comment by LinqLover** [comment]
I encountered the same issue with `pushd`. I think this line should just refer to SC3045 instead of SC3044?

https://github.com/koalaman/shellcheck/blob/ac8fb00504ed6da83fe5c5f83e72e4663ff6b439/src/ShellCheck/Checks/ShellSupport.hs#L398

**Comment by e-kwsm** [comment]
https://www.shellcheck.net/wiki/SC3044:

> In addition to <code>declare</code>, the following also cause <a
href="[https://github.com/koalaman/shellcheck/blob/v0.11.0/src/ShellCheck/Checks/ShellSupport.hs#L434-L436](view-source:https://github.com/koalaman/shellcheck/blob/v0.11.0/src/ShellCheck/Checks/ShellSupport.hs#L434-L436)">SC3044 failures as of v0.11.0</a>: <code>let</code>, <code>caller</code>, <code>builtin</code>, <code>complete</code>, <code>compgen</code>, <code>dirs</code>, <b><code>disown</code></b>, <code>enable</code>, <code>mapfile</code>, <code>readarray</code>, <code>pushd</code>, <code>popd</code>, <code>shopt</code>, <code>suspend</code>, and <code>typeset</code>.

(emphasis mine)
